spec/notifiers/new_applicant_notifier_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe NewApplicantNotifier, type: :mailer do
+  let(:mailer) { Struct.new(:deliver_later).new(nil) }
+  let(:job) { mock_model Job, owner: nil }
+  let(:job_user) { mock_model JobUser, job: job, user: nil }
+
+  it  'must work' do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.